test/indexer/test_hca_indexer.py

```diff
@@ -1777,6 +1777,36 @@ def test_related_files_field_exclusion(self):
                                      })
         self.assertEqual(0, hits["hits"]["total"])
 
+    def test_samples_downstream_entities(self):
+        bundle = self.bundle_fqid(uuid='79fa91b4-f1fc-534b-a935-b57342804a70',
+                                  version='2020-12-10T10:30:00.000000Z')
+        self._index_canned_bundle(bundle)
+        hits = self._get_all_hits()
+        entity_ids = {hit['_source']['entity_id'] for hit in hits}
+        expected_entity_ids = {
+            # Root bundle
+            '79fa91b4-f1fc-534b-a935-b57342804a70',
+            'abe1a013-af7a-45ed-8c26-f3793c24a1f4',
+            'c60bf214-0340-53c0-b75b-af84a82808d6',
+            # Stitched downstream samples
+            '0635494f-d45d-48a9-81b1-5f1d4ac15835',
+            '17aea4fd-cd72-4c36-8ded-d686162804b4',
+            '2cbca5ef-0850-4c4a-9d24-05454a1060e8',
+            '3f646bab-c762-4dbd-8f7b-b374427dac74',
+            '442f68c7-502a-4fc7-a3b8-94e9574cd22d',
+            '5fbd4bb4-e761-4345-99b8-bd5e141a8b94',
+            '8c0ef0ae-0687-4f2a-8a27-d833b7032a26',
+            '8c1b5489-7bf1-40c0-ba60-5e6b079e108a',
+            'a39685fd-b34a-4db7-b973-381919edb96e',
+            'c98d14ec-8761-45f7-8e86-e63c188bd97f',
+            'ddd10ecc-f7b7-4370-82cf-14e30987b751',
+            'ee2bfb40-7aa4-4c78-b78a-91aa69c7dc6e',
+            'f37a0dec-b7ba-4b34-8e51-732fb5ac7213',
+            'f47bb6a8-a753-4d40-8254-28fee9d03c1b',
+            'f5d590c5-b307-4b99-af98-8ddba1e2a266'
```

Comment:
This only proves that something is contributed to the upstream samples, but not what. The test should assert the expectation from the issue description. And if possible I would like the assertion to be less literal.